app/code/Magento/Payment/Helper/Data.php

```diff
@@ -150,6 +150,7 @@ public function getStoreMethods($store = null, $quote = null)
             $res[] = $methodInstance;
         }
 
+        // phpcs:ignore Generic.PHP.NoSilencedErrors
```

Comment:
Why? Just remove silencing operator

Reply:
Ping @p-bystritsky

Reply:
@orlangur but if someone extends on this and then will have some output on this point ?

Reply:
PHP Warnings should not be displayed in prod anyway, so, it won't cause any issue

Reply:
ok.

Reply:
@orlangur but seems like problem not on silencing operator.
![DeepinScreenshot_select-area_20190430152114](https://user-images.githubusercontent.com/2481206/56961316-2bf2b380-6b5c-11e9-8120-73ac2ef464bc.png)
![DeepinScreenshot_select-area_20190430152514](https://user-images.githubusercontent.com/2481206/56961317-2bf2b380-6b5c-11e9-85c4-f54d00c713e7.png)

Reply:
@Nazar65 this is something you env specific. Try fixing path and check vendor folder contents.

Reply:
@orlangur thanks :+1:  all done :heavy_check_mark: